src/index.html

```diff
@@ -12,7 +12,39 @@
       href="styles/index.scss"
     />
```

Comment:
The HTML file is linking to a `.scss` file. You should compile your SCSS into a CSS file and link to that CSS file instead. This is crucial for the stylesheet to be applied correctly.